gourmet/gtk_extras/LinkedTextView.py

```diff
@@ -97,22 +69,28 @@ def __init__ (self):
         self.connect('motion-notify-event',self.motion_notify_event)
         self.connect('visibility-notify-event',self.visibility_notify_event)
 
-    def make_buffer (self):
+    def make_buffer(self):
         return LinkedPangoBuffer()
 
-    # Links can be activated by pressing Enter.
-    def key_press_event(self, text_view, event):
+    def key_press_event(self,
+                        text_view: 'LinkedTextView',
+                        event: Gdk.Event) -> bool:
+        """Handle Enter key-press on time links."""
         keyname = Gdk.keyval_name(event.keyval)
-        if keyname in ['Return','KP_Enter']:
+        if keyname in ['Return', 'KP_Enter']:
             buffer = text_view.get_buffer()
-            iter = buffer.get_iter_at_mark(buffer.get_insert())
-            return self.follow_if_link(text_view, iter)
+            itr = buffer.get_iter_at_mark(buffer.get_insert())
```

Comment:
Should be `_, itr = ...`, as `get_iter_at_mark` returns a Tuple[bool, Optional[Gtk.TextIter]]`

Reply:
nope, not here, since the cursor is already within the box.